topovis/topologytoolkit/src/processors/morsesmalecomplex.cpp

```diff
@@ -56,9 +56,19 @@ const ProcessorInfo MorseSmaleComplex::processorInfo_{
 const ProcessorInfo MorseSmaleComplex::getProcessorInfo() const { return processorInfo_; }
 
 MorseSmaleComplex::MorseSmaleComplex()
-    : Processor(), inport_{"triangulation"}, outport_{"outport"} {
+    : Processor()
+    , inport_{"triangulation"}
+    , outport_{"outport"}
+    , returnSaddleConnectors_{"returnSaddleConnectors", "Return Saddle Connectors", false}
+    , computeSaddleConnectors_{"computeSaddleConnectors", "Compute Saddle Connectors", false}
+    , saddleConnectorsPersistenceThreshold_{"saddleConnectorsPersistenceThreshold_",
+                                            "Saddle Connectors Persistence Threshold", 0.0f, 0.0f,
+                                            100.0f} {
```

Comment:
I am guessing 100 is the max value of threshold. In TTK, maximum value of the persistence is actually dependent on the scalar data range (max_persistence = max_val - min_val). It would be ideal to make the max value dependent on scalar data range.

Same is the case in TopologicalSimplification here where max value is set as 1000: [Link](https://github.com/inviwo/modules/blob/eec934eb59ea28ba8f0b666a4bd5153c5c3c8568/topovis/topologytoolkit/src/processors/topologicalsimplification.cpp#L60)